src/libs/ReportUtils.js

```diff
@@ -1171,6 +1171,17 @@ function getDisplayNamesWithTooltips(personalDetailsList, isMultipleParticipantR
     });
 }
 
+/**
+ * Get the report given a reportID
+ *
+ * @param {String} reportID
+ * @returns {Object}
+ */
+function getReport(reportID) {
+    // Deleted reports are set to null and lodashGet will still return null in that case, so we need to add an extra check
+    return lodashGet(allReports, `${ONYXKEYS.COLLECTION.REPORT}${reportID}`, {}) || {};
```

Comment:
why do we prefer `{}` instead of `null` for missing reports? I think `null` or `undefined` is much more obvious for showing that a report is missing.

Reply:
https://github.com/Expensify/App/blob/653926dc89a72df6d9c557fe675840711e179dbb/src/libs/ReportUtils.js#L1320-L1323

Im not sure why this function is showing as a diff if it is already present in `ReportUtils.js` in `main`. Did you duplicate it?

Reply:
Ignore all my comments, I just see that you moved this!

Reply:
yes, no problem. Changing it to return `null` would definitely need updates everywhere this function is used and is totally out of the scope of this PR.